src/open_r1/evaluate.py

```diff
@@ -27,6 +27,27 @@
 from lighteval.utils.language import Language
 
 
+# Prompt template from simple-evals: https://github.com/openai/simple-evals/blob/6e84f4e2aed6b60f6a0c7b8f06bbbf4bfde72e58/math_eval.py#L17
+MATH_QUERY_TEMPLATE = """
```

Comment:
This is a good template as it makes less strong assumptions about whether the model has been trained / prompted to emit boxed outputs